app/policies/work_version_policy.rb

```diff
@@ -19,12 +19,21 @@ def edit?
   end
   alias_method :update?, :edit?
 
+  # Work versions cannot be destroyed if they haven't been persisted.
+  # Admins can delete a work version in any state, as long as it's the latest one
+  # Regular users can only delete draft versions that they can edit
   def destroy?
+    return false if record.new_record?
```

Comment:
Is this the same as `persisted? == false`

Reply:
Yup, `persisted?` and `new_record?` are inverses of each other